### PR TITLE
Extract gradle test into separate actions

### PR DESCRIPTION
**Pull request by glefloch**
As discussed in zulip (https://quarkusio.zulipchat.com/#narrow/stream/187038-dev/topic/Gradle.20in.20CI), the branch extract the gradle steps into separate action. 

@aloubyansky, @stuartwdouglas can you have a look?

### CODE REVIEW

File: .github/workflows/ci-actions.yml [hunk 1]
```diff
@@ -240,6 +231,51 @@ jobs:
           name: test-reports-linux-jvm${{matrix.java.name}}
           path: 'test-reports.tgz'
 
+  linux-gradle-jvm-tests:
+    name: JDK ${{matrix.java.name}} JVM Tests
```
Comment (aloubyansky) [inline .github/workflows/ci-actions.yml:235]: Should we mention Gradle in the name here?

File: .github/workflows/ci-actions.yml [hunk 2]
```diff
@@ -288,6 +315,37 @@ jobs:
           name: test-reports-windows-jdk11-jvm
           path: 'test-reports.tgz'
 
+  windows-gradle-jdk11-jvm-tests:
```
Comment (aloubyansky) [inline .github/workflows/ci-actions.yml:318]: Shouldn't it be simply gradle-jdk11-jvm-tests?

Comment (aloubyansky) [reply .github/workflows/ci-actions.yml:318]: Ah sorry. Ignore this one.

**Review by aloubyansky — APPROVED** [review]
Looks good to me.

File: .github/workflows/ci-actions.yml [hunk 3]
```diff
@@ -288,6 +315,37 @@ jobs:
...
+    name: Windows JDK 11 JVM Tests
```
Comment (aloubyansky) [inline .github/workflows/ci-actions.yml:319]: Same here, let's add "Gradle" to the name here.

File: .github/workflows/ci-actions.yml (hunk 1 above)
Comment (glefloch) [reply .github/workflows/ci-actions.yml:235]: I updated it to :
````
linux-jvm-gradle-tests:

  name: JDK ${{matrix.java.name}} JVM Gradle Tests
````

**Pull request merged**